test/components/variational_forms/test_uccsd_advanced.py

```diff
@@ -188,11 +188,9 @@ def test_uccsd_hf_qUCCSD(self):
 
         # initial state
         init_state = HartreeFock(
-            num_orbitals=fermionic_transformation.molecule_info['num_orbitals'],
-            qubit_mapping=fermionic_transformation._qubit_mapping,
-            two_qubit_reduction=fermionic_transformation._two_qubit_reduction,
+            num_spin_orbitals=fermionic_transformation.molecule_info['num_orbitals'],
             num_particles=fermionic_transformation.molecule_info['num_particles'],
-            sq_list=fermionic_transformation.molecule_info['z2_symmetries'].sq_list)
+            qubit_converter=self.qubit_converter)
```

Comment:
This will need to change since the idea was to test under different conditions. Here it was with tapering - maybe, depending how 'auto' is setup, this will have to change. I do not know whether it would be helpful to have a TODO comment in these or not - clearly they will need fixing up once the stack is altered and hopefully the use of the self.qubit_convertor is self evidently not what would be wanted since we want to change up the configurations that are tested

Reply:
Yes. I mainly had to change the unittests for the linters to be happy. All of the UCCSD-related unittests are disabled as of now. But once we merge this PR, I will replace them with new tests in #66 :+1: Thanks for pointing it out though!